students/surowiec_marek/lesson_01_basics/sum_of_three_numbers.py

```diff
@@ -5,5 +5,5 @@
 print('Podaj trzecia liczbe:')
 c = int(input())
 print('Suma wskazanych liczb wynosi:')
-s = a + b + c
-print(s)
\ No newline at end of file
+suma_liczb = a + b + c
+print(suma_liczb)
```

Comment:
PEP8 nakazuje pisać komentarze po angielsku. W mojej opinii jeżeli to możliwe to cały kod powinien być napisany po angielsku, żeby ułatwić ewentualne rozszerzanie przez innych ludzi, albo mainenance w innych stronach świata.
Nie poprawiaj, ale postaraj się o tym pamiętać.